crates/matrix-sdk/src/event_cache/linked_chunk/mod.rs

```diff
@@ -410,7 +410,11 @@ impl<const CAP: usize, Item, Gap> LinkedChunk<CAP, Item, Gap> {
     ///
     /// Because the `position` can be invalid, this method returns a
     /// `Result`.
-    pub fn remove_item_at(&mut self, position: Position) -> Result<Item, Error> {
+    pub fn remove_item_at(
+        &mut self,
+        position: Position,
+        empty_chunk: EmptyChunk,
```

Comment:
We should probably update the docs to let people know what the new argument is about. I know `EmptyChunk` contains docs for this as well, but I would expect a small sentence here as well.